src/index.html

```diff
@@ -10,13 +10,95 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
+
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="logo">
+          <a href="#section">
+            <img
+              src="./images/logo.png"
+              alt="Moyo Logo"
+            />
+          </a>
+        </div>
+
+        <nav>
+          <ul>
+            <li>
+              <a
+                class="box box-1 is-active"
+                href="#"
+              >
+                apple
+              </a>
+            </li>
+            <li>
+              <a
+                class="box box-2"
+                href="#"
+              >
+                samsung
+              </a>
+            </li>
+            <li>
+              <a
+                class="box box-3"
+                href="#"
+              >
+                smartphones
+              </a>
+            </li>
+            <li>
+              <a
+                class="box box-4"
+                href="#"
+              >
+                laptops & computers
+              </a>
+            </li>
+            <li>
+              <a
+                class="box box-5"
+                href="#"
```

Comment:
The `href` attribute in the `<a>` tag should not contain spaces. Consider using an anchor link like `#` without spaces.